ohapi/command_line.py

```diff
@@ -301,3 +305,22 @@ def upload(directory, metadata_csv, master_token=None, member=None,
                 mode=mode,
                 access_token=access_token,
             )
+
+
+@click.command()
+@click.option('-s', '--subject', help='subject', required=True)
+@click.option('-m', '--message_body', help='compose message', required=True)
+@click.option('-at', '--access_token', help='access token', required=True)
+@click.option('--all_members', help='all members',
+              default=False, show_default=True)
+@click.option('--project_member_ids', help='show project_member_id',
```

Comment:
`show project_member_ids` isn't correct, is it? I think it should be "list of project member ids to which a message will be sent" or along the lines. It would also be cool if the help text could inform people how the IDs should be separated (comma, space,…)